2022-06-17 10:54:43,875
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 2376, in <module>
    graph.move_figure(item, x - start_point[0],y - start_point[1])
TypeError: 'NoneType' object is not subscriptable
2022-06-17 10:55:12,857
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 2376, in <module>
    graph.move_figure(item, x - start_point[0],y - start_point[1])
TypeError: 'NoneType' object is not subscriptable
2022-06-17 10:56:00,113
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 542, in hlabel
    sg.Graph.draw_text(graph, msg.upper(), (x, y), font=fontstr)
AttributeError: 'NoneType' object has no attribute 'upper'
2022-06-17 10:56:01,433
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 542, in hlabel
    sg.Graph.draw_text(graph, msg.upper(), (x, y), font=fontstr)
AttributeError: 'NoneType' object has no attribute 'upper'
2022-06-17 10:57:33,395
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 2376, in <module>
    graph.move_figure(item, x - start_point[0],y - start_point[1])
TypeError: 'NoneType' object is not subscriptable
2022-07-20 09:22:06,506
2022-07-20 09:22:06,553
2022-07-20 09:22:06,553
2022-07-20 09:22:06,553
2022-07-20 09:22:06,568
2022-07-20 09:22:06,600
2022-07-20 09:22:06,615
2022-07-20 09:22:06,741
2022-07-20 09:22:06,741
2022-07-20 09:22:06,757
2022-07-20 09:22:06,757
2022-07-20 09:22:06,757
2022-07-20 09:22:06,772
2022-07-20 09:22:06,772
2022-07-20 09:22:06,772
2022-07-20 09:22:06,788
2022-07-20 09:22:06,803
2022-07-20 09:22:06,835
2022-07-20 09:22:06,850
2022-07-20 09:22:06,850
2022-07-20 09:22:06,866
2022-07-20 09:22:06,866
2022-07-20 09:22:06,866
2022-07-20 09:22:06,866
2022-07-20 09:22:06,881
2022-07-20 09:22:06,897
2022-07-20 09:22:06,897
2022-07-20 09:22:06,913
2022-07-20 09:22:06,913
2022-07-20 09:22:06,929
2022-07-20 09:22:06,929
2022-07-20 09:22:06,929
2022-07-20 09:22:06,929
2022-07-20 09:22:06,976
2022-07-20 09:22:06,992
2022-07-20 09:22:06,992
2022-07-20 09:22:06,992
2022-07-20 09:22:06,992
2022-07-20 09:22:07,007
2022-07-20 09:22:07,023
2022-07-20 09:22:07,038
2022-07-20 09:22:07,038
2022-07-20 09:22:07,038
2022-07-20 09:22:07,101
2022-07-20 09:22:07,116
2022-07-20 09:22:07,116
2022-07-20 09:22:07,132
2022-09-19 09:58:04,560
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 1181, in load_sketch_template
    with open(f"{filename}.pkl", "rb") as pk:
FileNotFoundError: [Errno 2] No such file or directory: 'C:/Users/Cr/Locatedraw/Locate-draw/BOROS LINK 33 TREES R.pkl.pkl'
2022-12-23 16:26:19,885
2022-12-23 16:26:19,901
2022-12-23 16:26:19,901
2022-12-23 16:26:32,589
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 792, in road
    TK.itemconfig(r, capstyle="tk.ROUND")
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 2898, in itemconfigure
    return self._configure(('itemconfigure', tagOrId), cnf, kw)
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 1627, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: bad cap style "tk.ROUND": must be butt, projecting, or round
2022-12-23 16:26:37,136
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 792, in road
    TK.itemconfig(r, capstyle="tk.ROUND")
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 2898, in itemconfigure
    return self._configure(('itemconfigure', tagOrId), cnf, kw)
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 1627, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: bad cap style "tk.ROUND": must be butt, projecting, or round
2022-12-23 16:27:13,791
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 950, in line
    TK.itemconfig(l, capstyle="TK.ROUND")
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 2898, in itemconfigure
    return self._configure(('itemconfigure', tagOrId), cnf, kw)
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 1627, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: bad cap style "TK.ROUND": must be butt, projecting, or round
2022-12-23 16:27:16,760
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 950, in line
    TK.itemconfig(l, capstyle="TK.ROUND")
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 2898, in itemconfigure
    return self._configure(('itemconfigure', tagOrId), cnf, kw)
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 1627, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: bad cap style "TK.ROUND": must be butt, projecting, or round
2022-12-23 16:27:36,466
Traceback (most recent call last):
  File "C:\Users\Cr\Locatedraw\Locate-draw\ezdrawlnew.py", line 838, in cable
    TK.itemconfig(cable, capstyle='tk.ROUND')
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 2898, in itemconfigure
    return self._configure(('itemconfigure', tagOrId), cnf, kw)
  File "C:\Users\Cr\AppData\Local\Programs\Python\Python38-32\lib\tkinter\__init__.py", line 1627, in _configure
    self.tk.call(_flatten((self._w, cmd)) + self._options(cnf))
_tkinter.TclError: bad cap style "tk.ROUND": must be butt, projecting, or round
